packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -288,7 +290,7 @@ To exchange the authorization code for tokens, pass the code to your authorizati
 **Example request**
 
 ```bash
-curl --location --request POST \
+curl --request POST \
   --url https://${yourOktaDomain}/oauth2/default/v1/token \
```

Comment:
Let's use the same format for the data in the body in this request as the below token exchange request, using `--data-urlencode`.